pkg/sql/delegate/show_functions.go

```diff
@@ -0,0 +1,83 @@
+// Copyright 2022 The Cockroach Authors.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.txt.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0, included in the file
+// licenses/APL.txt.
+
+package delegate
+
+import (
+	"fmt"
+
+	"github.com/cockroachdb/cockroach/pkg/sql/lexbase"
+	"github.com/cockroachdb/cockroach/pkg/sql/opt/cat"
+	"github.com/cockroachdb/cockroach/pkg/sql/sem/catconstants"
+	"github.com/cockroachdb/cockroach/pkg/sql/sem/tree"
+)
+
+// delegateShowFunctions implements SHOW FUNCTIONS which returns all the
+// user-defined functions.
+// Privileges: None.
+//
+//	Notes: postgres does not have a SHOW FUNCTIONS statement.
+func (d *delegator) delegateShowFunctions(n *tree.ShowFunctions) (tree.Statement, error) {
+	flags := cat.Flags{AvoidDescriptorCaches: true}
+	_, name, err := d.catalog.ResolveSchema(d.ctx, flags, &n.ObjectNamePrefix)
+	if err != nil {
+		return nil, err
+	}
+	// If we're resolved a one-part name into <db>.public (which is the behavior
```

Comment:
FWIW, I saw it says `Compatibility with CockroachDB v1.1: use D.public.T instead.` But I agree with the override you're doing here.

Reply:
ah, i had copied this comment over from `show_tables.go`, but yes that sounds right